src/web3_wrapper.ts

```diff
@@ -34,9 +34,14 @@ export class Web3Wrapper {
             return undefined;
         }
     }
+    public toWei(ethAmount: BigNumber.BigNumber): BigNumber.BigNumber {
+        const balanceWei = this.web3.toWei(ethAmount, 'ether');
+        return balanceWei;
+    }
     public async getBalanceInEthAsync(owner: string): Promise<BigNumber.BigNumber> {
```

Comment:
Yeah, that makes sense to me. This was a legacy method ported from OTC. I'll refactor it to return wei.